engine/runtime/src/main/java/org/enso/interpreter/node/expression/debug/EvalNode.java

```diff
@@ -86,7 +86,7 @@ RootCallTarget parseExpression(LocalScope scope, ModuleScope moduleScope, String
     var sco = newInlineContext.localScope().getOrElse(LocalScope::root);
     var mod = newInlineContext.getModule();
     var m = org.enso.interpreter.runtime.Module.fromCompilerModule(mod);
-    var toTruffle = new IrToTruffle(context, src, m.getScope(), compiler.getConfig());
+    var toTruffle = new IrToTruffle(context, src, m.getScopeBuilder(), compiler.getConfig());
```

Comment:
I can imagine that this is a place where `EvalNode` wants to incrementally add something to the scope. Still having some sequence like:
```
var b = m.newScopeBuilder(reuseEverything=true)
b.registerXyz(...)
b.build() // and install into `Module`
```
is a way better than having a getter to existing scope builder.